test/models/project_test.rb

```diff
@@ -55,7 +55,7 @@ def clone_repository(project)
     it "defaults to release number 1" do
```

Comment:
I don't think the description matches the test anymore... maybe "increments release number"?